cumulusci.yml

```diff
@@ -341,6 +453,23 @@ tasks:
                 - "UNABLE_TO_LOCK_ROW"
                 - "connection was cancelled here"
             retry_always: True
+    
+    disable_triggers:
+        class_path: cumulusci.tasks.salesforce.trigger_handlers.SetTDTMHandlerStatus
+        description: "Disable all NPSP Trigger Handlers"
+        options:
+            active: False
```

Comment:
I'm not really sure its necessary to use a restore file for these. You could just have one set to Inactive and the other to Active. NPSP does not have inactive trigger handlers by default  anyway.

Reply:
I need some of them to be on, for this data set - we didn't persist all the CampaignMembers, for example, that the NPSP triggers automatically create. The restore file makes it run faster because it cuts the number of API calls.